src/main/groovy/org/ajoberstar/gradle/git/release/semver/NearestVersionLocator.groovy

```diff
@@ -37,87 +38,97 @@ import org.slf4j.LoggerFactory
  * @since 0.8.0
  */
 class NearestVersionLocator {
-	private static final Logger logger = LoggerFactory.getLogger(NearestVersionLocator)
+    private static final Logger logger = LoggerFactory.getLogger(NearestVersionLocator)
+    private static final Version unknown = Version.valueOf('0.0.0')
```

Comment:
Since `Version`s are immutable, this should follow normal constant naming conventions (all caps).